packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -497,72 +480,56 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         view
         returns (bytes memory)
     {
-        // TODO: Consider adding BridgePool address to the ancillary data packet.
-        // TODO: Consider hashing all of the params that can be compared against the L2 Deposit contract into a single
-        // "relay ancillary data hash" to reduce storage costs.
-        bytes memory intermediateAncillaryData = "";
-
-        // Add data inferred from the original deposit on L2:
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "chainId",
-            uint256(_depositData.chainId)
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "depositId",
-            uint256(_depositData.depositId)
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueAddress(
-            intermediateAncillaryData,
-            "l1Recipient",
-            _depositData.l1Recipient
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueAddress(
-            intermediateAncillaryData,
-            "l2Sender",
-            _depositData.l2Sender
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "amount",
-            _depositData.amount
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "slowRelayFeePct",
-            uint256(_depositData.slowRelayFeePct)
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "instantRelayFeePct",
-            uint256(_depositData.instantRelayFeePct)
-        );
-        intermediateAncillaryData = AncillaryData.appendKeyValueUint(
-            intermediateAncillaryData,
-            "quoteTimestamp",
-            uint256(_depositData.quoteTimestamp)
-        );
+        return
+            AncillaryData.getKeyValueBytes32(
```

Comment:
To maintain consistency with the rest of AncillaryDataLib I think you should rename this function to "append..."

Reply:
Good point. Will try making it append and see if that costs much gas.

Reply:
Making this an "append" method costs about 400 gas (200 for relay and 200 for settle). Thoughts?

Reply:
I added this, but we might reconsider re-removing the append if we want to make this a bit cheaper.

Reply:
Yeah i guess if there's no starting ancillary data to append to, then no need to append! Let's revert the append.